src/NodaTime.Demo/OffsetDemo.cs

```diff
@@ -62,6 +62,58 @@ public void ConstructionFromTimeSpan()
             Assert.AreEqual(5400, offset.Seconds);
         }
 
+        [Test]
+        public void Add()
+        {
+            var lefHandOffset = Offset.FromHours(5);
+            var rightHandOffset = Offset.FromHours(6);
+
+            var result = Snippet.For(Offset.Add(lefHandOffset, rightHandOffset));
+
+            Assert.AreEqual(result, lefHandOffset + rightHandOffset);
+        }
+
+        [Test]
+        public void Substract()
+        {
+            var lefHandOffset = Offset.FromHours(7);
+            var rightHandOffset = Offset.FromHours(5);
+
+            var result = Snippet.For(Offset.Subtract(lefHandOffset, rightHandOffset));
+
+            Assert.AreEqual(result, lefHandOffset - rightHandOffset);
+        }
+
+        [Test]
+        public void CompareTo()
+        {
+            var smallerOffset = Offset.FromHours(3);
+            var largerOffset = Offset.FromHours(5);
+            var equalOffset = Offset.FromHours(5);
+
+            var lessThan = smallerOffset.CompareTo(largerOffset);
```

Comment:
We need Snippet.For on just one of these calls.

Reply:
```
            var lessThan = smallerOffset.CompareTo(largerOffset);
            var equalTo = largerOffset.CompareTo(Snippet.For(Offset.FromHours(5)));
            var largerThan = largerOffset.CompareTo(smallerOffset);
```

Is this what is required? Just getting sure. Does the same follows for the method Equals, which currently looks like this:

```
 public void Equals()
        {
            var equalOffset1 = Offset.FromHoursAndMinutes(1, 30);
            var equalOffset2 = Offset.FromHoursAndMinutes(1, 30);
            var inequalOffset = Offset.FromHours(2);

            var isEqual = equalOffset1.Equals(equalOffset2);
            var isInequal = equalOffset1.Equals(inequalOffset);

            Assert.True(isEqual);
            Assert.False(isInequal);
        }
```

Reply:
Yes - basically the purpose of `Snippet.For` is just to tell the snippet tooling "please include this snippet with the specified member" - so it's only needed once. But it *is* needed once - that Equals doesn't have any.